test/blackbox-tests/test-cases/reason/ppx/jbuild

```diff
@@ -0,0 +1,6 @@
+(jbuild_version 1)
+
+(library
+ ((name reasonppx)
+  (wrapped false)
```

Comment:
You need to add `(kind ppx_rewriter)` so that the library is linked with `-linkall`. Otherwise, it won't be linked into the custom ppx driver

Reply:
Oops, that's a bit embarrassing. I pushed a fix.